src/main/java/com/walmartlabs/x12/util/SegmentUtil.java

```diff
@@ -0,0 +1,51 @@
+package com.walmartlabs.x12.util;
+
+import com.walmartlabs.x12.X12Segment;
+
+import java.util.List;
+
+public class SegmentUtil {
+
+    private List<X12Segment> segmentLines;
+    private int currentSegmentIdx;
+
+    public SegmentUtil(List<X12Segment> segmentLines) {
+        currentSegmentIdx = 0;
+        this.segmentLines = segmentLines;
+    }
+
+    /**
+     * compare the current segment index and see if
+     * there are any more segment lines after it
+     */
+    public boolean hasMoreSegmentLines() {
+        return (currentSegmentIdx < segmentLines.size()) ;
```

Comment:
Extra brackets

Reply:
subjective... I like the extra brackets as it quickly wraps the expression visually...